tests/rptest/tests/rpk_acl_test.py

```diff
@@ -71,8 +71,7 @@ def test_modify_then_query_error(self):
                 FailureSpec(FailureSpec.FAILURE_ISOLATE,
                             self.redpanda.controller()))
 
-            # Timeout must be larger then hardcoded timeout of 5s within redpanda
-            _ = superclient.acl_list(request_timeout_overhead=30)
+            _ = superclient.acl_list()
```

Comment:
Won't this set this test back to 5s timeout, which may cause it to fail?

Reply:
I.e., don't we need to set `timeout` on the rpk object to 30s now in this test.

Reply:
No not really because the default timeout is 30s anyway and the `request_timeout_overhead` thing now effectively applies to everything so the special casing in `acl_list` isn't needed anymore.